liferay-echo/liferay-portal
# LPD-31828 Update text for Theme CSS Client Extension on Design tab regarding frontend token definitions

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-31828)

## Changes

Change the text for the CSS client extension section.

## Test Scenario 1

1. Go to Pages
2. Click on the kebab menu in the top right > Configuration
3. Navigate to the design tab and check the text for the CSS client Extension.

![image](https://github.com/user-attachments/assets/c4abc5fb-a2ce-4fa2-b2ad-cf64255814e8)


> [!NOTE]
>No test needed for this.
